<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GostEl</class>
 <widget class="QWidget" name="GostEl">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Размеры по ГОСТ H=0,25D</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>H, мм</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="sgostel_cb"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="diagostel_cb"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="Hgostel_le"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Толщина, мм</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Диаметр, мм</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pbGostElOK">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbGostElCancel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;h&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;, мм&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="h1gostel_le"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
